umh-core/pkg/config/env.go

```diff
@@ -105,6 +106,16 @@ func LoadConfigWithEnvOverrides(ctx context.Context, configManager *FileConfigMa
 				FSMInstanceConfig: FSMInstanceConfig{
 					DesiredFSMState: "active", // Default desired state for Redpanda
 				},
+				RedpandaServiceConfig: redpandaserviceconfig.RedpandaServiceConfig{
+					Topic: redpandaserviceconfig.TopicConfig{
+						DefaultTopicRetentionMs:    604800000,
+						DefaultTopicRetentionBytes: 0,
```

Comment:
(suggestion) Shouldn't we have a number here, atleast a big number like a `1 GB` ? Because an high throughput topic could quickly fill up the disk and with the retention policy of 7 days, the application would be already down with low disk.

Reply:
We currently have no limit (per-default) on mgmtcompanion, so i would leave it as it for now.